db/migrations/20210517101603_create_song_file_table.ts

```diff
@@ -0,0 +1,15 @@
+import { Knex } from "knex";
+
+export async function up(knex: Knex): Promise<void> {
+  return knex.schema.createTable("song_file", (table) => {
```

Comment:
Can a file belong to multiple songs? If not, consider dropping this table in favour of having a `song_id` column on the `file` table.